### PR TITLE
Home page headerr

### PR DESCRIPTION
**Pull request by marwaghassa**
New Home page header branch resolving issue #1

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -30,7 +30,8 @@ module.exports = {
         "2xs": "5rem",
       },
       boxShadow: {
-        xl: "-15px -15px #7EAC8A",
+        xl: "-40px -35px #7EAC8A",
```
Comment (ziadbaroudi) [inline tailwind.config.js:33]: changing xl values will cause problems in other components, please don't override existing values, instead create a new one with a different name, xxl for example. 
Component using xl: cart-item.

**Review by ziadbaroudi — CHANGES_REQUESTED** [review]

**Review by osamaakb — CHANGES_REQUESTED** [review]
it doesn't look like the figma design,

image

**Comment by osamaakb** [comment]
Will close this for now

**Comment by marwaghassa** [comment]
Home Page Header styled

**Comment by osamaakb** [comment]
please resolve the conflicts first

**Comment by osamaakb** [comment]
it is still not 100% like figma, but good job @marwaghassa , I will merge it for the sake of time

**Review by osamaakb — APPROVED** [review]

**Pull request merged**